test/src/test/java/hudson/model/AbstractProjectTest.java

```diff
@@ -435,23 +435,6 @@ public void addTriggerDifferentType() throws Exception {
     @Test
     public void configDotXmlSubmissionToDifferentType() throws Exception {
         TestPluginManager tpm = (TestPluginManager) j.jenkins.pluginManager;
```

Comment:
This variable is now unused. Unused variables are generally considered bad programming practice by most style guides, so I suggest deleting it (along with the corresponding import):

```suggestion
```

This makes the code easier to read by avoiding an unused variable.